src/Filter/Uglifyjs.php

```diff
@@ -50,8 +64,33 @@ class Uglifyjs extends AssetFilter
     public function output($filename, $input)
     {
         $cmdSep = $this->_settings['version'] <= 1 ? ' - ' : '';
-        $cmd = $this->_settings['node'] . ' ' . $this->_settings['uglify'] . $cmdSep . $this->_settings['options'];
         $env = array('NODE_PATH' => $this->_settings['node_path']);
+        $cmd =
+            $this->_settings['node'] . ' ' .
+            $this->_settings['uglify'] . $cmdSep .
+            $this->_settings['options'];
+
+        if ($this->_settings['create_map']) {
```

Comment:
This should probably have an `isset()` check or it will emit notices when the setting is not defined.